node-gui/backend/src/messages.rs

```diff
@@ -149,11 +150,35 @@ pub struct SendDelegateToAddressRequest {
     pub delegation_id: String,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Transaction {
```

Comment:
Doesn't `Transaction` look confusing? Perhaps `SignedTransaction` would be better?